scripts/pycairoinstall.py

```diff
@@ -4,48 +4,48 @@
 import urllib.request
 
 if 'Windows' in platform.system():
-    # In case the python version is 3.6 and the system is 32-bit, try pycairo‑1.19.1‑cp37‑cp37m‑win32.whl version of cairo
+    # In case the python version is 3.6 and the system is 32-bit, try pycairo‑1.19.2‑cp37‑cp37m‑win32.whl version of cairo
     if sys.version[:3]=='3.6' and platform.machine()=='x86':
-        urllib.request.urlretrieve("https://download.lfd.uci.edu/pythonlibs/w3jqiv8s/pycairo-1.19.1-cp36-cp36m-win32.whl", "pycairo-1.19.1-cp36-cp36m-win32.whl")
-        os.system("pip install pycairo-1.19.1-cp36-cp36m-win32.whl")
-        os.remove("pycairo-1.19.1-cp37-cp37m-win32.whl")
+        urllib.request.urlretrieve("https://github.com/naveen521kk/pycairo/releases/download/v1.19.2/pycairo-1.19.2-cp36-cp36m-win32.whl", "pycairo-1.19.2-cp36-cp36m-win32.whl")
```

Comment:
**Question :** Why do you use your personal fork and not the official repo?

Reply:
The problem with official one is that they not yet provide it.